eve/single_node/single_node.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+
+dd if=/dev/zero of=/var/lib/kube_lvm bs=1M count=20480
```

Comment:
`truncate -s 2G /var/lib/kube_lvm` is likely faster and introduces less needless IO (could also use `dd` to create a sparse file, but I always forget what the right arguments for that are :smile:)